lib/logstash/filters/dns.rb

```diff
@@ -244,11 +254,13 @@ def retriable_getaddress(name)
 
   private
   def getname(address)
-    @resolv.getname(address).force_encoding(Encoding::UTF_8)
+    name = @resolv.getname(address).force_encoding(Encoding::UTF_8)
```

Comment:
is this ```force_encoding(Encoding::UTF_8)``` part really necessary, would the magic header do this work also? just curious to be sure.